keel-core/src/main/kotlin/com/netflix/spinnaker/keel/pause/ActuationPauser.kt

```diff
@@ -66,59 +69,55 @@ class ActuationPauser(
   fun pauseApplication(application: String) {
     log.info("Pausing application $application")
     pausedRepository.pauseApplication(application)
-    publisher.publishEvent(ApplicationActuationPaused(application))
+    publisher.publishEvent(ApplicationActuationPaused(application, "Application $application paused", clock))
   }
 
   fun resumeApplication(application: String) {
     log.info("Resuming application $application")
     pausedRepository.resumeApplication(application)
-    publisher.publishEvent(ApplicationActuationResumed(application))
-    resourceRepository.getResourcesByApplication(application)
-      .forEach {
-        // helps a user not be confused by an out of date status from before a pause
-        publisher.publishEvent(ResourceActuationResumed(it))
-      }
+    publisher.publishEvent(ApplicationActuationResumed(application, "Application $application resumed", clock))
   }
 
   fun pauseResource(id: String) {
     log.info("Pausing resource $id")
     pausedRepository.pauseResource(id)
-    publisher.publishEvent(ResourceActuationPaused(resourceRepository.get(id)))
+    publisher.publishEvent(ResourceActuationPaused(resourceRepository.get(id), null, clock.instant()))
```

Comment:
What's the `null` there?

It would also be nice to be consistent about how the exception constructors use `Clock` vs `Instant`. The calls just above to `ApplicationActuationPaused` are passing in the clock, and here it's passing an instant. I don't have a strong preference (lean toward the latter) but it would be good if they always did the same thing.

Reply:
The `null` is for `reason`. It was already `null` previously but the new constructor required it. As for clock vs. instant, I agree. The reason this case was "special" was that I was once creating synthetic `ResourceActuationPaused` events from an `ApplicationActuationPaused` by copying the timestamp. I can undo this now that we're just using the original application events.